app/views/initiatives/_form.html.erb

```diff
@@ -18,7 +18,10 @@
    
     <%= f.label :description, 'Description', class: 'form-field-label' %>
     <%= f.text_area :description, maxlength: '5000', placeholder: 'Add a description here...', class: 'textarea w-input' %>
-   
+    <div class="init-footer">
+    <%= f.label :completion, 'Completed', class: 'form-field-label' -%>
+    <%= f.check_box :completion, :class => "w-checkbox", :style => "margin: 5px" %>
+      </div>
```

Comment:
Can you indent this so that the divs are aligned, and the label and checkbox are nested one indent to the right of the div tags? It will be easier to read.

Reply:
I've added indentation